evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response, NextFunction } from 'express';
+import { driverService } from '../services';
+
+// Example for controller
+export const getDrivers = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const page = req.params.page || 1;
+  const limit = req.params.limit || 10;
+
+  try {
+    const drivers = await driverService.getDrivers({}, page, limit);
+    return res.status(200).json({
```

Comment:
To be consistence with all services, lets create a JsonResponse object that will have the below params and return it

Reply:
Just an idea for now

Reply:
Nice 👍🏽